lib/client.js

```diff
@@ -322,7 +337,8 @@ export default class Client {
         tags: ['origin:' + originHostname]
       })
 
-      console && console.error('Invalid domain: ' + this._origin)
+      clearDocument()
+      appendErrorToBody(this._origin)
```

Comment:
⛏ super tiny thought, you can pass full string here instead of origin to make this function generic.

Reply:
Done!